.babelrc.js

```diff
@@ -9,5 +9,16 @@ if (process.env.NODE_ENV !== `test`) {
 module.exports = {
   sourceMaps: true,
   presets: ["babel-preset-gatsby-package"],
+  overrides: [
+    {
+      test: "**/*.ts",
+      plugins: [
+        [
+          "@babel/plugin-transform-typescript",
+          { isTSX: true, allExtensions: true },
```

Comment:
Question: Is the `allExtensions`useful if we glob for `*.ts`? :)

Reply:
i suppose not! kept fiddling with these options to try and get things to work and forgot what was actually important. Removed!